velox/core/QueryCtx.h

```diff
@@ -145,7 +145,14 @@ class QueryCtx {
 class ExecCtx {
  public:
   ExecCtx(memory::MemoryPool* pool, QueryCtx* queryCtx)
-      : pool_(pool), queryCtx_(queryCtx), vectorPool_{pool} {}
+      : pool_(pool),
+        queryCtx_(queryCtx),
+        enableOperatorBufferCache_(
+            !queryCtx || queryCtx->queryConfig().enableOperatorBufferCache()) {
+    vectorPool_ = enableOperatorBufferCache_
```

Comment:
We could put vectorPool_ in initializer list? Thanks!

Reply:
I intentionally put it in the body to avoid relying on the initialization order of vectorPool_ and enableOperatorBufferCache_. I've moved it to initializer list. Thanks!